test/unit/org/apache/cassandra/repair/FuzzTestBase.java

```diff
@@ -183,7 +183,9 @@ public static void setUpClass()
         UnitConfigOverride.maybeOverrideConfig();
 
         DatabaseDescriptor.daemonInitialization();
-        DatabaseDescriptor.setCommitLogWriteDiskAccessMode(Config.DiskAccessMode.mmap);
+
+        if (DatabaseDescriptor.getCommitLogWriteDiskAccessMode() == Config.DiskAccessMode.direct)
```

Comment:
nit:  this warrants a comment explaining the limitations this imposes on testing compression, encryption, etc.  particularly because this is a static, impacting every test running in the same jvm…